README.md

```diff
@@ -1,21 +1,36 @@
-The Game of Life
-================
+# The Game Of Life
+
 Rendering engine for watching Life patterns evolve, created with Python 2.6 and
 PyGame.
 
 * Implements the HashLife algorithm, so patterns can be viewed at super-linear
 speeds with moderate requirements.
 * Can load basic .lif and .cells files, for easy use.
 
-Controls:
---------
+## Dependencies
+* Python 2 (2.6 or 2.7)
+* Pygame (`pip install pygame`)
+
+## Installation
+1. Run the following commands :
+```
+sudo git clone https://github.com/EricBurnett/GameOfLife.git
```

Comment:
`git clone` should not require sudo - can that be removed?

Reply:
That depends the area, I often had an issue with that because of being non-root user

Reply:
But it can be removed in fact I tried in another area on my hard drive and it works

Reply:
Merging as-is - thanks for validating; I can remove the `sudo` post-merge :).